### PR TITLE
Strict function / block prototypes

### PR DESCRIPTION
**Pull request by adam-b**
Use strict prototypes in function / block declarations.
(-Wstrict-prototypes)

### CODE REVIEW

**Comment by adam-b** [comment]
I signed it!

File: EarlGrey/Synchronization/GREYRunLoopSpinner.m [hunk 1]
```diff
@@ -123,14 +123,14 @@ - (void)grey_drainRunLoopInActiveModeForDrains:(NSUInteger)exitDrainCount {
...
-- (BOOL)grey_drainRunLoopInActiveModeAndCheckCondition:(BOOL (^)())stopConditionBlock
+- (BOOL)grey_drainRunLoopInActiveModeAndCheckCondition:(BOOL (^)(void))stopConditionBlock
                                                forTime:(CFTimeInterval)time {
   NSString *activeMode = [self grey_activeRunLoopMode];
   CFRunLoopTimerRef wakeUpTimer = [self grey_setupWakeUpTimerInMode:activeMode];
   __block BOOL conditionMet = NO;
   __weak __typeof__(self) weakSelf = self;
 
-  void (^beforeSourcesConditionCheckBlock)() = ^{
+    void (^beforeSourcesConditionCheckBlock)(void) = ^{
```
Comment (khandpur) [inline EarlGrey/Synchronization/GREYRunLoopSpinner.m]: formatting is off

File: EarlGrey/Synchronization/GREYRunLoopSpinner.m [hunk 2]
```diff
@@ -143,7 +143,7 @@ - (BOOL)grey_drainRunLoopInActiveModeAndCheckCondition:(BOOL (^)())stopCondition
     }
   };
 
-  void (^beforeWaitingConditionCheckBlock)() = ^{
+    void (^beforeWaitingConditionCheckBlock)(void) = ^{
```
Comment (khandpur) [inline EarlGrey/Synchronization/GREYRunLoopSpinner.m]: formatting is off

File: EarlGrey/Synchronization/GREYRunLoopSpinner.m [hunk 3]
```diff
@@ -78,14 +78,14 @@ - (void)grey_drainRunLoopInActiveModeForDrains:(NSUInteger)exitDrainCount {
...
-  void (^drainCountingBlock)() = ^{
+    void (^drainCountingBlock)(void) = ^{
     drainCount++;
     if (drainCount >= exitDrainCount) {
       CFRunLoopStop(CFRunLoopGetCurrent());
     }
   };
 
-  void (^wakeUpBlock)() = ^{
+    void (^wakeUpBlock)(void) = ^{
```
Comment (khandpur) [inline EarlGrey/Synchronization/GREYRunLoopSpinner.m]: formatting is off

File: EarlGrey/Synchronization/GREYRunLoopSpinner.m [hunk 4]
```diff
@@ -78,14 +78,14 @@ - (void)grey_drainRunLoopInActiveModeForDrains:(NSUInteger)exitDrainCount {
   }
 
   __block NSUInteger drainCount = 0;
-  void (^drainCountingBlock)() = ^{
+    void (^drainCountingBlock)(void) = ^{
```
Comment (khandpur) [inline EarlGrey/Synchronization/GREYRunLoopSpinner.m]: formatting is off

**Comment by adam-b** [comment]
Fixed indents.

**Pull request merged**